ngraph/frontend/onnx_import/include/onnx_import/core/graph.hpp

```diff
@@ -46,6 +46,7 @@ namespace ngraph
             const std::string& get_name() const { return m_graph_proto->name(); }
             OutputVector make_ng_nodes(const Node& onnx_node) const;
             const GraphCache& get_graph_cache() const;
+            std::vector<int64_t> get_opset_versions() const;
```

Comment:
I am not sure if set more than one opset version is real scenario, but it is provided by ONNX standard. Any comments?

Reply:
Could you prepare a syntetic model with multiple opsets and run verification from ONNX library?

Reply:
Since opset version is a map you should not return a vector of int IMHO.
```// Operator Sets
//
// OperatorSets are uniquely identified by a (domain, opset_version) pair.
message OperatorSetIdProto {
  // The domain of the operator set being identified.
  // The empty string ("") or absence of this field implies the operator
  // set that is defined as part of the ONNX specification.
  // This field MUST be present in this version of the IR when referring to any other operator set.
  optional string domain = 1;

  // The version of the operator set being identified.
  // This field MUST be present in this version of the IR.
  optional int64 version = 2;
}
```

Reply:
@tsocha I checked such model with `onnx.checker.check_model` and it seems to be accepted. In case of domain you are right it should be also passed from parent graph. I've fixed it.